app/src/main/java/org/mozilla/tv/firefox/webrender/cursor/CursorEventRepo.kt

```diff
@@ -44,11 +47,13 @@ class CursorEventRepo(screenController: ScreenController) {
         data class CursorMoved(val direction: Direction) : CursorEvent()
```

Comment:
`CursorEventRepo.webRenderDirectionEvents` is duplicating existing logic: it's filtering the events to make sure they're direction-events, filtering action down, mapping them to the appropriate direction, filtering if the WebRenderFragment is active, etc. However, we already have that logic in order to translate the cursor in response to a direction key.

Technically, scroll events happen in response to Cursor move events so I think it makes more sense to push scroll events in response to these cursor move events, i.e. around the call site of `mutatePosition`. This would also remove the duplication and, I think, clarify the code. In fact, I think that means we won't need the `CursorEventRepo` at all.

Reply:
It hadn't occurred to me, but you're right.  That's definitely a a pretty glaring oversight on my part.  I'll merge the two.